tools/gulp/constants.ts

```diff
@@ -29,6 +29,12 @@ export const HTML_MINIFIER_OPTIONS = {
   removeAttributeQuotes: false
 };
 
+/** Packages that will be published to NPM by the release task. */
+export const RELEASE_PACKAGES = [
+  'cdk',
+  'material',
+];
```

Comment:
This way you can find stuff better and it can be like a config. It's probably the same question about having a configuration file for Pixacto.